pages/community/precice-workshop-2022.md

```diff
@@ -0,0 +1,66 @@
+---
+title: preCICE workshop 2022
+keywords: precice workshop 2022, event, events, workshop
+summary:
+permalink: precice-workshop-2022.html
+toc: false
+redirect_from: /preCICE2022/
+---
+
+<img class="img-responsive center-block" src="images/events/precice2022.svg" alt="preCICE Workshop banner" style="max-width: 500px; margin:auto;">
+
+The 3rd preCICE Workshop will be held virtually through [SimTech](https://www.simtech.uni-stuttgart.de/) and the [University of Stuttgart](https://www.uni-stuttgart.de/en/), from February 21-24, 2022. The workshop is a coming together of the preCICE community to share ideas, experiences and knowledge about using preCICE, and to learn from others in the process.
+
+[Registration](https://precice.org/precice-workshop-2022.html#registration) is now open. We look forward to welcoming back previous participants, and introducing more members to the preCICE community.
+
+Due to the uncertain times we are experiencing with the ongoing COVID-19 pandemic and the great success of the virtual workshop in February 2021, we are planning to host the workshop virtually again.
+
+If you have any questions regarding the workshop, please contact us at [`precice2022@ipvs.uni-stuttgart.de`](mailto:precice2022@ipvs.uni-stuttgart.de).
+
+## Program
+
+Coming soon.
+
+## Registration
+
+This is an academic, non-profit conference. However, we still have to collect a small registration fee.
+For early registrations (until January 17), the registration fee is
+
+* 50€ for members of universities and public research institutes and
+* 150€ for all other participants.
+
+Registration is still possible after this date, with a registration fee of
+
+* 100€ for members of universities and public research institutes and
+* 300€ for all other participants.
+
+Please use the [registration form](*link*) to register.
```

Comment:
@IshaanDesai we can merge once we have the registration link to put it in here (and in the call for contribution section)

Reply:
Okay yes. I will update here as soon as I have the link, should be soon.